erpnext/public/js/utils/serial_batch_selector.js

```diff
@@ -43,6 +43,8 @@ erpnext.stock.SerialBatchSelector = Class.extend({
 			serial_no: this.has_serial_no ? this.item.serial_no : null,
 		}
 
+		this.load_all_batches(preselect=false);
```

Comment:
Loading will work fine but there's a problem:
We want the row `selected_qty` to be set from what we already have selected in the transaction.

If I've selected Batch A x5 and Batch B x10 and haven't selected Batch C x0
Then the table should show `selected_qty` for Batch A=5, Batch B=10, Batch C=10

Also keep in mind that frappe.call() is asynchronous. So you can either use this.load_all_batches.then(...) OR you can pass an optional callback function to `load_all_batches` that will be called before updating the UI to update `selected_qty` values